go.mod

```diff
@@ -61,13 +35,13 @@ require (
 )
 
 replace (
-	// Result of go get github.com/containerd/containerd@v0.2.x
-	github.com/containerd/containerd => github.com/containerd/containerd v0.2.10-0.20180716142608-408d13de2fbb
-	github.com/docker/docker => github.com/openshift/moby-moby v1.4.2-0.20190308215630-da810a85109d
+	github.com/containerd/containerd => github.com/containerd/containerd v1.3.6
+	github.com/docker/docker => github.com/docker/docker v0.0.0-20190404075923-dbe4a30928d4
 	github.com/docker/libnetwork => github.com/docker/libnetwork v0.8.0-dev.2.0.20171107005402-dcf79f8c7fcb
 	github.com/godbus/dbus => github.com/godbus/dbus v0.0.0-20181101234600-2ff6f7ffd60f
-	github.com/moby/buildkit => github.com/dmcgowan/buildkit v0.0.0-20170731200553-da2b9dc7dab9
 	github.com/opencontainers/runtime-tools => github.com/opencontainers/runtime-tools v0.8.0
+	// CVE-2020-9283 fix
+	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
```

Comment:
We pull in the vulnerable ssh code, therefore we need to bump this to a patched version.